test/e2e/date-select.test.js

```diff
@@ -0,0 +1,49 @@
+import * as dateFns from 'date-fns'
+
+import parsers from '../../common/parsers'
+
+import { fillInForm } from './_helpers'
+import { pmuUser } from './_roles'
+import {
+  home,
+  incomingMoves,
+  outgoingMoves,
+  allocations,
+  singleRequests,
+} from './_routes'
+import { page, movesDashboardPage } from './pages'
+
+const jumpToDate = async (t, paginatedPage, value) => {
+  const expectedDate = dateFns.format(parsers.date(value), 'yyyy-MM-dd')
+
+  await t.navigateTo(outgoingMoves)
+  await t.click(movesDashboardPage.nodes.pagination.dateSelectLink)
+  await fillInForm([
+    {
+      selector: page.nodes.dateSelectInput,
+      value,
+    },
+  ])
+  await page.submitForm()
+  await t.expect(page.getCurrentUrl()).contains(expectedDate)
+}
+
+fixture('Jump to date').beforeEach(async t => {
```

Comment:
I've manually tested this work with the new population route, and it does. I'm still working on the population E2E tests, so I'll add a test in here once that's ready.